shopping-cart/shopping-cart.js

```diff
@@ -1,15 +1,23 @@
 import cards from '../data/cards.js';
-import cart from '../data/cart.js';
 
 import findById from '../findById.js';
 
```

Comment:
It's a little more readable if you group these together without the white space.